UnitsNet.Tests/GeneratedCode/AccelerationTestsBase.g.cs

```diff
@@ -327,5 +327,18 @@ public void UnitsDoesNotContainUndefined()
         {
             Assert.DoesNotContain(AccelerationUnit.Undefined, Acceleration.Units);
         }
+
+        [Fact]
+        public void AllUnitsHaveAtLeastOneAbbreviationSpecified()
+        {
+            var units = Enum.GetValues(typeof(AccelerationUnit)).Cast<AccelerationUnit>();
+            foreach(var unit in units)
+            {
+                if(unit == AccelerationUnit.Undefined)
+                    continue;
+
+                var defaultAbbreviation = UnitAbbreviationsCache.Default.GetDefaultAbbreviation(unit);
```

Comment:
Just a thought, would `Default` be better named `SystemDefault` or `CurrentCulture` or something?
It's heavily based on `CultureInfo.CurrentCulture` and `CultureInfo.CurrentUICulture` behind the scenes.